### DISCUSSION TITLE
part-ii/2018-20/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#259] [Question comments]
Part II 2018 Asymptotic Methods: Paper 3, Section II, B 

https://questions.tripos.org/part-ii/2018-20/

**Comment by acornnroca** [comment]
<details><summary>Solution for part a</summary>

curves of steepest descent on sin q cosh p -1 = 0. Forms pitchfork shape, including bit along negative real axis.



</details>
